bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/channelhelper/CameraChannelHelper.java

```diff
@@ -36,8 +37,8 @@
 public class CameraChannelHelper extends ChannelHelper {
     private static final String QUALITY_CONF_ENTRY = "quality";
     private static final String LIVE_PICTURE = "/live/snapshot_720.jpg";
-    private boolean isLocal;
-    private @Nullable String vpnUrl;
+
+    private @NonNullByDefault({}) String vpnUrl;
```

Comment:
Please avoid bypassing the null checks. I would prefer to keep a `@Nullable` for `vpnUrl`.

Reply:
ok, changed